### PR TITLE
Hpt-436/repoint-google-analytics-repairs-online

### PR DESCRIPTION
**Pull request by adamtry**
[HPT-436](https://hackney.atlassian.net/jira/software/projects/HPT/boards/116?selectedIssue=HPT-436) Integrate Google Analytics into Repairs Online

This had already been done, but the tag was out of date and the site cannot pull new environment variables from the lambda function env variables without a redeploy.

A new Google Tag Manager account was created and configured in the code here which will be referenced from the existing Google Analytics account to allow for fine-grained analytics.

I removed the env var and reference to the SSM parameter because as has been proven this is redundant and potentially misleading - we can't change it without redeployment.

[HPT-436]: https://hackney.atlassian.net/browse/HPT-436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: yarn.lock
Comment (Duslerke) [inline yarn.lock:1]: I would have done these minor package upgrades as a separate PR so that if something actually does break, that the rollback wouldn't need to rollback the analytics tags :thinking: 

But now that it's here, we can give it a go and see how it goes once deployed.

**Review by Duslerke — APPROVED** [review]
Yeah, the 90 day re-run window we have doesn't exactly fit the needs for such rarely -changing public variable updating.

File: pages/_app.js [hunk 1]
```diff
@@ -27,16 +27,14 @@ function MyApp({ Component, pageProps, err  }) {
...
-
   return <>
     <Script id="google-tag-manager" strategy="afterInteractive">
       {`
         (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
         new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
         j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
         'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','${gtmToken}');
+        })(window,document,'script','dataLayer','GTM-WGX2SMF2');
```
[the remark this answers is not in the document]
Comment (adamtry) [reply pages/_app.js:37]: Not at all - it's public
